index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Website</title>
+  <link rel="stylesheet" href="style/style.css">
+</head>
+<body>
+    <section class="main">
+        <header class="header">
+            <div class="header__logo">
+                <img src="/images/favicon.png" alt="Nova">
+            </div>
+            <nav class="header__nav">
+                <ul class="nav__list">
+                    <li class="nav__item"><a href="TheTeam">The Team</a></li>
+                    <li class="nav__item"><a href="Pricing">Pricing</a></li>
+                    <li class="nav__item"><a href="Features">Features</a></li>
+                </ul>
+            </nav>
+        </header>
+    </section>
+
+    <section class="main-content">
+        <div class="main__content">
+            <h1>Beautiful Free Nova Template</h1>
+            <p>A top-notch premium quality template for your next web project.</p>
+            <button>Download FREE!</button>
+        </div>
+        <div class="main__image">
+            <img src="/images/screen.png" alt="Next Gen App">
+        </div>
+    </section>
+
+    <section class="features">
+        <h2>Features</h2>
+        <ul class="features__container">
+            <li class="feature__item">
+                <img src="/images/responsive.svg" alt="Responsive">
+                <h3>Fully Responsive</h3>
+                <p>Looks amazing on any device: smartphone, tablet, laptop, and desktop.</p>
+            </li>
+            <li class="feature__item">
+                <img src="/images/customizable.svg" alt="Customizable">
+                <h3>Customizable</h3>
+                <p>Change the colors, pictures, or any of the sections to suit your needs.</p>
+            </li>
+            <li class="feature__item">
+                <img src="/images/design.svg" alt="Design">
+                <h3>Slick and Beautiful Design</h3>
+                <p>Trendy and fresh design, fits any website.</p>
+            </li>
+        </ul>
+    </section>
+    <div class="features__logos">
+        <img src="/images/logos.png" alt="logos">
+    </div>
+    <section class="image-section">
+        <div class="content-section">
+            <div class="image">
+                <img src="/images/imac.png" alt="iMac" />
+            </div>
+            <div class="text">
```

Comment:
Tutaj wrzuciłbym znacznik `article` zamiast `div` - bo to faktycznie jakaś konkretna treść.